FLEDGE_extended_PA_reporting.md

```diff
@@ -73,7 +73,7 @@ We consider the scenario where a buyer wants to learn the click through rate of
 been in an interest group for a given time.
 
 The buyer may implement `getImpressionReportBucket()` and `getClickReportBucket()` which map an
-interest group and the time the user has spent in that interest group to a 128-bit integer.
+interest group and the time the user has spent in that interest group, calculated using [interest group recency](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#:~:text=The%20recency%20field%20is%20duration%20of%20time%20(in%20minutes)%20from%20when%20this%20device%20joined%20this%20interest%20group%20until%20now%2C%20using%20the%20noising%20and%20bucketing%20scheme), to a 128-bit integer.
```

Comment:
1. You're linking to `recency` in `reportWin()` which is heavily bucketed and noised, but this example code is in `generateBid()` which has access to the unnoised and unbucketed `browserSignals.recency`.  We could change it to link to the description in the `generateBid()` part of the explainer but that kinda points at the `reportWin()` piece.  Perhaps we should just say `browserSignals.recency` and not link to the explainer.
2. This explainer was published prior to adding/supporting `browserSignals.recency` which points to the fact that callers of `joinAdInterestGroup()` can already calculate recency themselves (e.g. by storing the time of joining the IG in the `userBiddingSignals`), so they don't have to use `browserSignals.recency`.  I guess it's good to encourage people to use `browserSignals.recency` now that it's available.

Reply:
Sounds good.  I've updated a little bit more and expanded on the example description.  Let me know how it looks.  

@alexmturner Would you be able to take a look at this also?

Reply:
LGTM!